WebApiCoreSeed.WebApi/Controllers/ValuesController.cs

```diff
@@ -1,45 +1,53 @@
-﻿using System;
+
+namespace WebApiCoreSeed.WebApi.Controllers
+{
+
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Authorization;
 
-namespace WebApiCoreSeed.WebApi.Controllers
-{
+
     [Route("api/[controller]")]
     public class ValuesController : Controller
     {
         // GET api/values
         [HttpGet]
         public IEnumerable<string> Get()
         {
+            //TODO: Add Functionality GetData from database
             return new string[] { "value1", "value2" };
         }
 
         // GET api/values/5
         [HttpGet("{id}")]
         public string Get(int id)
         {
+            //TODO: Add Functionality GetData from database
             return "value";
         }
 
         // POST api/values
         [HttpPost]
         public void Post([FromBody]string value)
         {
+            //TODO: Add Functionality Insert data in to the database

```

Comment:
Minor grammar suggestion: "into" should be a single word. (It also applies to the rest of the comments in this file)